packages/models-library/src/models_library/generics.py

```diff
@@ -65,10 +64,3 @@ class Envelope(GenericModel, Generic[DataT]):
     @classmethod
     def parse_data(cls, obj):
         return cls.parse_obj({"data": obj})
-
-    @validator("data", pre=True)
-    @classmethod
-    def empty_dict_is_none(cls, v):
```

Comment:
Instead of removing this which could have consequences for other services, I would look at the api for the webserver. If I understood you correctly you are doing this because [this](https://github.com/bisgaard-itis/osparc-simcore/blob/c11504d503dd71feec6c9dc544e97b64e74509c5/api/specs/web-server/_projects_ports.py#L48) endpoint in the webserver returns a `{"data":{}}` when the project has no outputs. However, as we can see from that openapi specification it should actually return a `Envelope({})` (which should serialize to `{"data":null}` if I am not mistaken). I.e. I think the real problem here is that the webserver implementation and its specification do not agree. Hence, I suggest to fix this. 
To do this you could either ensure the webserver endpoint really returns a `Envelope[dict[NodeID, ProjectOutputGet]]` as its specification dictates. However, this might break something in the frontend which relies on the webservers implementation. Hence, I would suggest you correct the openapi-specification by creating a pydantic model which correctly corresponds to the webservers implementation and use this one both on the webserver-side as well as on the api-server side.